tests/Propel/Tests/Runtime/ActiveQuery/CriteriaTest.php

```diff
@@ -1122,10 +1122,10 @@ public function testWithSimpleLock()
     {
         $c = new Criteria();
         $c->withLock(Lock::SHARED);
-        $this->assertInstanceOf(Lock::class, $c->getLock(), 'withLock() adds a shared read lock to the Criteria');
-        $this->assertEquals(Lock::SHARED, $c->getLock()->getType());
-        $this->assertEmpty($c->getLock()->getTableNames());
-        $this->assertFalse($c->getLock()->isNoWait());
+        self::assertInstanceOf(Lock::class, $c->getLock(), 'withLock() adds a shared read lock to the Criteria');
+        self::assertEquals(Lock::SHARED, $c->getLock()->getType());
+        self::assertEmpty($c->getLock()->getTableNames());
+        self::assertFalse($c->getLock()->isNoWait());
```

Comment:
this was actually correct, see also their docs.
In general, you can use assertSame though for scalar value comparison better.

Reply:
According to the [docs](https://phpunit.readthedocs.io/en/7.5/assertions.html) it's only a matter of personal preference but I can revert this. I've changed it because I had hints from inspection system in PHPStorm.

Regarding assertEquals vs assertSame - yes, I really don'y know whether to conform to how tests are currently written (not to introduce anything more besides the needs for this PR) or to write them how I personally feel (to pass inspections, properly typehinted etc.) 😄

Reply:
reverted `self::` to `$this->`